js/main.js

```diff
@@ -54,6 +52,157 @@ class UI {
     el.parentNode.removeChild(el);
   }
 }
+class Navigation {
+  constructor() {
+    this.navs = document.querySelectorAll('[href]');
+  }
+
+  init() {
+    this.navs.forEach((nav) => {
+      nav.addEventListener('click', (e) => {
+        if (e.target.tagName === 'A') {
+          this.changeTabs(e);
+          this.changeContent(e);
+        }
+      });
+    });
+  }
+
+  changeTabs(e) {
+    this.navs.forEach((nav) => nav.classList.remove('active'));
+    e.target.classList.add('active');
+  }
+
+  changeContent(e) {
+    document.querySelectorAll('.content-box').forEach((item) => {
+      item.classList.remove('active');
+    });
+    const selector = e.target.getAttribute('href');
+    const content = document.querySelector(selector);
+    content.classList.add('active');
+  }
+}
+class Layout {
+  constructor() {
+    this.app = document.getElementById('app');
+    this.app.classList.add('layout');
+    this.header = document.createElement('header');
+    this.header.setAttribute('id', 'header');
+    this.header.classList.add('header', 'position-fixed', 'top-0', 'w-100');
+    this.header.innerHTML = `<nav class="navbar px-3">
+                               <div class="page-title">
+                                 <a href="#list" class="page-title text-white active">Awesome Book</a>
+                               </div>
+                               <ul class="nav-list d-flex mt-3">
+                                 <li id="position-relative" >
+                                   <span class="active-navlink"></span>
+                                 </li>
+                                 <li class="nav-item " >
+                                   <a href="#list" class="ps-3 text-white navLink active">List</a>
+                                 </li>
+                                 <li class="nav-item" >
+                                   <a href="#form" class="ps-3 text-white navLink">Add Book</a>
+                                 </li>
+                                 <li class="nav-item" >
+                                   <a href="#contact" class="ps-3 text-white navLink">Contact</a>
+                                 </li>
+                               </ul>
+                             </nav>`;
+    const today = new Date();
```

Comment:
- [ ] It would be better if you use `Luxon` library to display the current date below the navigation bar. Because that makes it easier to work with dates and times in JavaScript which also you will do in your future job.

Reply:
Thank You for your review